sdk/keyvault/azure-security-keyvault-administration/README.md

```diff
@@ -130,6 +130,137 @@ To update the value of any of the the available settings, we will call the Updat
 Setting updatedSetting
    = settingsClient.UpdateSetting(settingsList.Value[0].Name, options).Value;
 ```
+## Creating a BackupRestoreClient
+
+To create a new `BackupRestoreClient` to perform these operations, you need the endpoint to an Azure Key Vault HSM and credentials.
+
+Key Vault BackupRestoreClient client for C++ currently supports any `TokenCredential` for authenticating.
+
+```cpp Snippet:SampleBackupRestoreCreateCredential
+  auto credential
+      = std::make_shared<Azure::Identity::DefaultAzureCredential>();
+```
+
+Then, in the sample below, you can set `keyVaultUrl` based on an environment variable, configuration setting, or any way that works for your application.
+
+```cpp Snippet:SampleAdministration2CreateClient
+  // create client
+  BackupRestoreClient client(std::getenv("AZURE_KEYVAULT_HSM_URL"), credential);
+```
+## Create the SasTokenParameter
+
+Since these operations require a blob storage for the backup/restore operations, a SAS token is required for the connection between the services(Key Vault and Storage).  
+
+In this sample we rely on a couple of extra environment variables. 
+
+```cpp Snippet:SampleAdministration2CreateSASParam
+ SasTokenParameter sasTokenParameter;
+ // the backup/restore needs a SAS token to access the storage account
+ sasTokenParameter.Token
+     = Azure::Core::_internal::Environment::GetVariable("AZURE_KEYVAULT_BACKUP_TOKEN");
+ // the backup/restore needs a url to a blob storage resource
+ sasTokenParameter.StorageResourceUri
```

Comment:
Is there a way of using the backup APIs *without* a SAS token? Or is the SAS token parameter mandatory to use the API?

If the SAS token is not mandatory, it would be nice if the first "how to use" example was one without the SAS token, with the SAS token usage included in a sample.

Reply:
they can be used with managed identity, only i am unable to use that for development purposes so i can't (won't) write samples that i am unable to use. it's gambling and i don't do that

Reply:
sas is required for this thing here, it is a required param for the storage connection, not my first choice but there we have it. it is in the samples